jquery-editable-table.js

```diff
@@ -271,41 +271,40 @@ $.fn.editableTable = function (options) {
                             def: columnDef
                         });
                     }
-                })
+                });
+
                 columnsToAdd.sort((a, b) => a.order - b.order).forEach((colToAdd, index) => {
                     let newCell;
                     if (colToAdd.value !== null)
                         newCell = $(`<td>${colToAdd.value}</td>`);
                     else
                         newCell = $(`<td data-is-null></td>`);
-                        
-                   	// Apply any classes
+
+                    // Apply any classes
                     if (colToAdd.def.classes !== undefined && colToAdd.def.classes.length) {
                         colToAdd.def.classes.forEach(classToAdd => newCell.addClass(classToAdd));
                     }
 
-										// Apply any style
+                    // Apply any style
                     if (colToAdd.def.style !== undefined && colToAdd.def.style.length) {
                         newCell.attr("style", newCell.attr("style") + "; " + colToAdd.def.style);
                     }
-                    
+
                     // Hide if hidden
                     if (colToAdd.def.isHidden !== undefined && colToAdd.def.isHidden) {
                         newCell.hide();
-		            }
-			
-		            // Add to the column
-                    newRow.append(newCell);
+                    }
 
                     // Trigger any events
                     let columnDef = options.columns.filter(col => col.name === colToAdd.prop)[0];
                     if (typeof columnDef.afterAdd == 'function') {
                         columnDef.afterAdd(colToAdd.value, newCell);
                     }
-                });
 
-            }
-            else {
+                    // Add to the column
+                    newRow.append(newCell);
```

Comment:
Yes sure, but don't know anymore why. 😄 
Maybe to modify the newCell before add 🤔

Reply:
Hmmm if you're not sure of the reason, would you mind please putting the `.append` back before the `afterAdd` handler and check to see if your changes still work? I just already know of some personal usage that will break otherwise - unless it was required for your change